app/templates/content/framework-crown-hosting.html

```diff
@@ -0,0 +1,276 @@
+{% from 'macros/_breadcrumb.html' import breadcrumb %}
+{% extends "_base_page.html" %}
+
+{% block breadcrumb %}
+  {{ breadcrumb(crumbs) }}
+{% endblock %}
+
+{% block main_content %}
+
+    <div id="publication" class="html-publications-show">
+
+    <header class="publication-header" id="contents"><div class="inner-block floated-children">
+    <div class="headings">
+    <p class="document-type">Digital Marketplace guidance</p>
+    <h1>Crown Hosting Data Centres framework</h1>
+    </div>
+    <nav class="in-page-navigation"><h2>Contents</h2>
+    <ol>
+      <li><a href="#what-are-frameworks">What are frameworks?</a></li>
+      <li><a href="#frameworks-on-the-digital-marketplace">Frameworks on the Digital Marketplace</a></li>
+      <li><a href="#the-crown-hosting-data-centres-framework">The Crown Hosting Data Centres framework</a></li>
+      <li><a href="#further-information">Further information</a></li>
+    </ol></nav>
+    </div>
+    </header>
+    <div class="block publication-content">
+    <div class="inner-block floated-children">
+    <div class="govspeak">
+
+    <h2 id="what-are-frameworks">
+    <span class="number">1. </span>What are frameworks?</h2>
+    <p>
+      Government frameworks are agreements between the government and suppliers. The basic terms of use are agreed by both parties following a formal <a href="http://www.ojeu.eu/" rel="external">Official Journal of the European Union (OJEU)</a> procurement process. This saves public sector organisations and suppliers the time and cost traditionally associated with individual procurement contracts. Buyers and suppliers still need to sign a ‘call-off contract’ for each service procured through a framework.
```

Comment:
Are external links supposed to open in a new window/tab?  Or it just indicates that it's a different site?